repo_policy_compliance/blueprint.py

```diff
@@ -223,8 +247,11 @@ def health() -> Response:
         client = github_client.get()
         client.get_repo("canonical/repo-policy-compliance")
     except exceptions.InputError as exc:
-        return Response(response=str(exc), status=500)
+        return Response(response=str(exc), status=http.HTTPStatus.INTERNAL_SERVER_ERROR)
```

Comment:
Should the input error be map to 400 bad request?

Reply:
In this case `InputError` is raised if the `GITHUB_TOKEN` environment variable is not set. So probably this should be mapped to configuration error

Reply:
I'll fix this on the next PR

Reply:
In that case, it's a internal server error, up to you, thanks!